Kconfig

```
@@ -10,6 +10,13 @@ config CONFIGURATION_FILE_NAME
 	help
 		Configuration file name
 
+config DOCKER_FILE_NAME
+	string "Dockerfile name"
+	default "ubuntu16df"
```

For ease of maintenance, I'd like to use only the OS name, not the version. What do you think?
```suggestion
	default "ubuntu"
```

---

@t25kim I think that we must specify the version, as there may be other versions. For example, Ubuntu 18.04 or Alpine 3.7
Moreover, I think that we have to specify also sub-version for example Alpine 3.7: `alpine3.7` or `alpine3_7`. But these are only for the templates that are stored in repository, Users can save their with any names.

Tell me what do you think about it? And give examples, please!

Thank you!

---

Thank you for sharing your thought. 
Your idea is the best if possible. However is it possible to maintain all versions with limited manpower? Just thinking of Ubuntu, there are several versions such as 16, 18, 20, and next one. Also how long will we support for each version? 
If we manage only one representative version for each OS, it is okay to me to attach a version. But maintaining all the versions doesn't make sense, and it's better to develop other features at that time.

---

@t25kim I do not mind let's simplify. When it is necessary then and add.

---

Done!